Tests/Marketplace/marketplace_services.py

```diff
@@ -629,6 +629,33 @@ def _load_pack_dependencies(self, index_folder_path, first_level_dependencies, a
 
         return dependencies_data_result
 
+    def _get_updated_changelog_entry(self, changelog: dict, version: str, release_notes: str = None,
+                                     version_display_name: str = None, build_number_with_prefix: str = None,
+                                     released_time: str = None):
+        """
+
+        Args:
+            changelog (dict): The changelog from the production bucket.
+            version (str): The version that is the key in the changelog of the entry wished to be updated.
+            release_notes (str): The release notes lines to update the entry with.
+            version_display_name (str): The version display name to update the entry with.
+            build_number_with_prefix(srt): the build number to modify the entry to, including the prefix R (if present).
+            released_time: The released time to update the entry with.
+
+        """
+
+        changelog_entry = changelog[version]
```

Comment:
Let's be more careful here as we might want to reuse this function in the future. Please check whether the version is in the changelog and if not raise an exception.